yt_dlp/extractor/crunchyroll.py

```diff
@@ -692,9 +718,45 @@ def _real_extract(self, url):
         }
 
 
-class CrunchyrollBetaIE(CrunchyrollBaseIE):
+class CrunchyrollBetaBaseIE(CrunchyrollBaseIE):
+    params = None
+
+    def _get_params(self, lang):
+        if not CrunchyrollBetaBaseIE.params:
+            initial_state, app_config = self._get_beta_embedded_json(
+                self._download_webpage(f'https://beta.crunchyroll.com/{lang}', None,
+                                       note='Retrieving main page'), None)
```

Comment:
that's some weird indenting
```suggestion
            initial_state, app_config = self._get_beta_embedded_json(self._download_webpage(
                f'https://beta.crunchyroll.com/{lang}', None, note='Retrieving main page'), None)
```